--- a/drivers/usb/serial/option.c	2016-06-23 12:02:29.000000000 +0800
+++ b/drivers/usb/serial/option.c	2016-12-19 10:02:50.489469477 +0800
@@ -651,6 +651,8 @@
 };
 
 static const struct usb_device_id option_ids[] = {
+	{ USB_DEVICE(0x2c7c, 0x0125) },
+	{ USB_DEVICE(0x05c6, 0x9215) },
 	{ USB_DEVICE(OPTION_VENDOR_ID, OPTION_PRODUCT_COLT) },
 	{ USB_DEVICE(OPTION_VENDOR_ID, OPTION_PRODUCT_RICOLA) },
 	{ USB_DEVICE(OPTION_VENDOR_ID, OPTION_PRODUCT_RICOLA_LIGHT) },
